### PR TITLE
Revert "Update the docs and the framework"

### PR DESCRIPTION
**Pull request by adzhoudev**
This PR is to reverts commit 644344c45d88072ce35262baa9752ec0689b94b2, which is unintended and modified changelog.md and several files by running `3rd Party Cocoapods Release TEST ONLY` action in lan iOS SDK [repo](https://github.com/linkedin/audience-network-ios/actions/workflows/3P-public-release-test-only.yml) , revert it back to provide correct log information.

### CODE REVIEW

**Review by rjguzman42 — APPROVED** [review]
LGTM

File: docs/data/tags.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
 {"version":"0.0.3","date":"1660857042"},
 {"version":"0.1.0","date":"1664218848"},
 {"version":"0.1.1","date":"1664218798"},
+{"version":"0.1.3-internal-internal","date":"1706654601"},
```
Comment (heguanyu) [inline docs/data/tags.json:7]: don't add these 2

Comment (adzhoudev) [reply docs/data/tags.json]: thanks @heguanyu , I have updated this tags file.

**Review by heguanyu — APPROVED** [review]
LGTM

**Review by rjguzman42 — APPROVED** [review]

**Pull request merged**